### PR TITLE
Task/86 markdown description

### PR DESCRIPTION
**Pull request by zsoltker**

### CODE REVIEW

File: src/components/detailsPage/pages/resources/ResourcesDetailsPage.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  useEffect(() => {
+    const fetchAndSetSelfDescriptions = async () => {
+      try {
+        const response = await cypherQuery.getOneSelfDescriptions(resourceId);
+        console.log('Fetched data: ', response);
+        if (response) {
+          setSelfDescriptionData(response);
+        }
+      } catch (error) {
+        console.error('Error fetching self descriptions:', error);
```
Comment (devbysp) [inline src/components/detailsPage/pages/resources/ResourcesDetailsPage.tsx:28]: The error is already logged inside the `cypherQuery` you do not needed to do it again.

File: src/components/detailsPage/pages/resources/ResourcesMainContent.tsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+import styles from './ResourcesMainContent.module.css';
+
+export interface IDetailsCard extends ISelfDescription {
+}
+
+interface IDetailsCardProps {
+    cardData: IDetailsCard;
+}
+
+export default function ResourcesMainContent({ cardData }: Readonly<IDetailsCardProps>) {
```
Comment (devbysp) [inline src/components/detailsPage/pages/resources/ResourcesMainContent.tsx:18]: The name should be in singular `ResourceMainContent`. You just show the content of a single resource.

File: src/components/detailsPage/pages/resources/ResourcesDetailsPage.tsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+import { useContext, useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { CypherQueryApiService as cypherQuery } from 'services/cypherQueryApiService';
+
+import car from '../../../../assets/car.gif';
+
+import styles from './ResourcesDatails.module.css';
+import ResourcesMainContent from './ResourcesMainContent';
+
+export default function ResourcesDetailsPage() {
```
Comment (devbysp) [inline src/components/detailsPage/pages/resources/ResourcesDetailsPage.tsx:13]: The name should be in singular `ResourceDetailsPage`. You just show the details of a single resource.

**Review by devbysp — CHANGES_REQUESTED** [review]

File: src/common/markdown/GMarkdown.tsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+import styles from './GMarkdown.module.css';
+
+type GMarkdownProps = {
+    children: React.ReactNode;
+    isCard?: boolean;
+    maxContentHeight?: number;
+};
+
+const GMarkdown: React.FC<GMarkdownProps> = ({ isCard, maxContentHeight = 200, children }) => {
```
[the remark this answers is not in the document]
Comment (devbysp) [reply src/common/markdown/GMarkdown.tsx]: This is not solved.

File: src/common/markdown/Markdown.tsx [hunk 5]
```diff
@@ -0,0 +1,34 @@
+import MarkdownToJSX from 'markdown-to-jsx';
+import React, { useEffect, useRef, useState } from 'react';
+import './Markdown.css'
+
+type GMarkdownProps = {
```
Comment (devbysp) [inline src/common/markdown/Markdown.tsx:5]: Can you please rename the `GMarkdownProps` too.

**Review by devbysp — CHANGES_REQUESTED** [review]

**Review by devbysp — APPROVED** [review]

**Review by Huber-David — APPROVED** [review]

**Pull request merged**